examples/idempotency/src/using_redis_client_with_local_certs.py

```diff
@@ -9,7 +11,7 @@
     RedisCachePersistenceLayer,
 )
 
-redis_values: Any = parameters.get_secret("redis_info", transform="json")  # (1)!
+redis_values: dict[str, Any] = parameters.get_secret("redis_info", transform="json")  # (1)!
 
 
 redis_client = Redis(
```

Comment:
host and port must have default values other than `None`.

```
host=redis_values.get("REDIS_HOST", "localhost"),
port=redis_values.get("REDIS_PORT", 6379),
```

https://github.com/redis/redis-py/blob/master/redis/client.py#L176